src/cf-buttons/usage.md

```diff
@@ -1,11 +1,20 @@
-- [Default button](#default)
-- [Secondary button](#secondary)
-- [Destructive action button](#destructive)
-- [Disabled button](#disabled)
-- [Super button](#super)
+- [Atoms](#atoms)
+  - [Default button](#default-button)
+  - [Secondary button](#secondary-button)
+  - [Destructive action button](#destructive-action-button)
+  - [Disabled button](#disabled-button)
+  - [Super button](#super-button)
+  - [Button links](#button-links)
+  - [Icon buttons](#icon-buttons)
```

Comment:
@jimmynotjim and I are going to go over it on Monday in person. Things like these are really easy to go back and forth on for hours, so we'll try to come to a decision before the workshop so folks can have more time writing code and getting a feel for it before they start debating tricky cases. Especially, since I'm sure this won't be the last one. ;)